import "session_event.proto";

message Event_UserLeft {
    extend SessionEvent {
        optional Event_UserLeft ext = 1008;
    }
    optional string name = 1;
}
